topdown/http.go

```diff
@@ -124,6 +124,16 @@ func builtinHTTPSend(bctx BuiltinContext, args []*ast.Term, iter func(*ast.Term)
 
 		result = ast.NewTerm(obj)
 	}
+
+	// Cache for decision logging, if cache enabled/exists.
+	if bctx.NDCache != nil {
```

Comment:
Can we share code here with the inter/intra query cache handling? Does it use the same key?